model/jpa/src/main/java/org/jboss/aerogear/unifiedpush/jpa/dao/impl/JPAPushMessageInformationDao.java

```diff
@@ -141,6 +141,13 @@ public long getNumberOfPushMessagesForApplications() {
                 "IN (select p.pushApplicationID from PushApplication p)", Long.class).getSingleResult();
     }
 
+    @Override
+    public PushMessageInformation getPushMessageInformation(String id) {
+        return getSingleResultForQuery(createQuery(
+                "select pmi from PushMessageInformation pmi where pmi.id = :id")
+                .setParameter("id", id));
+
```

Comment:
good find, but even better would be to replace it with [`find(id)`](https://github.com/aerogear/aerogear-unifiedpush-server/blob/d216bfa1a9a4fe9a9165874cec0ed79b79ab80d6/model/jpa/src/main/java/org/jboss/aerogear/unifiedpush/jpa/dao/impl/JPABaseDao.java#L60) so this entire method could be removed